addons/binding/org.openhab.binding.seneye/README.md

```diff
@@ -63,4 +67,16 @@ A manual configuration through a `demo.items` file could look like this:
 String mySeneye_Temperature  "Temp [%s] C"        { channel="seneye:seneye:mySeneye:temperature" }
 String mySeneye_PH           "PH [%s]"            { channel="seneye:seneye:mySeneye:ph" }
 String mySeneye_NH3          "NH3 [%s]"           { channel="seneye:seneye:mySeneye:nh3" }
+```
+
+The sitemap could look like this:
+
+```
+sitemap home label="My home" {
+    Frame label="Aquarium" {
+        Text item=mySeneye_Temperature label="Temperature [%.1f °C]" icon="temperature"
+        Text item=mySeneye_PH3 label="PH [%.1f]" icon="water"
```

Comment:
You do not have the `3` in the name inside you `.items` file. I think it makes sense to align them so the user has a first working full example.

Reply:
UI sitemap matches items file

Reply:
No I doesn't:
`.items` file: `String mySeneye_PH` vs `.sitemap`: `Text item=mySeneye_PH3`

Reply:
ok, fixed (PH3 in the tank would be bad for the fish btw)